azext_edge/tests/edge/adr/test_namespace_devices_int.py

```diff
@@ -374,12 +374,18 @@ def test_namespace_device_lifecycle_operations(require_init, tracked_resources:
     assert endpoint_name_custom not in result
 
     # Remove endpoints (REST cert-auth, SSE username-auth, MQTT anonymous)
-    result = run(
-        f"az iot ops ns device endpoint inbound remove --device {device_name_2} "
-        f"--instance {instance_name} -g {resource_group} "
-        f"--endpoint {endpoint_name_onvif} {endpoint_name_media} {endpoint_name_rest} "
-        f"{endpoint_name_sse} {endpoint_name_mqtt} -y"
-    )
+    try:
+        result = run(
+            f"az iot ops ns device endpoint inbound remove --device {device_name_2} "
+            f"--instance {instance_name} -g {resource_group} "
+            f"--endpoint {endpoint_name_onvif} {endpoint_name_media} {endpoint_name_rest} "
+            f"{endpoint_name_sse} {endpoint_name_mqtt} -y"
+        )
+    except CLIInternalError as e:
+        # TODO - disable once bug is fixed / test is passing
```

Comment:
maybe in a next iteration we can think of a means to configure pytest.xfail in a common way that is less local to the test impl

Reply:
I honestly hesitated to use xfail at all 🍉 But you bring up a good point, would be a lot safer if we had one defined location for known flaky tests rather than sprinkling stuff like this around the codebase and hoping we remove them all one day 🙃